cohorts/variant_stats.py

```diff
@@ -158,6 +160,30 @@ def maf_somatic_variant_stats(variant, variant_metadata):
         normal_stats = _maf_variant_stats(variant, variant_metadata, prefix="n")
     return SomaticVariantStats(tumor_stats=tumor_stats, normal_stats=normal_stats)
 
+def _vcf_is_strelka(variant_file):
+    """Return True if variant_file given is in strelka format
+    """
+    return "strelka" in variant_file.lower()
+
+def _vcf_is_maf(variant_file):
+    """Retrun True if variant_file given is in .maf format
+    """
+    return ".maf" in variant_file.lower()
+
+def _vcf_is_mutect(variant_file):
+    """Return True if variant_file give is in mutect format
+    """
+    if "mutect" in variant_file.lower():
+        return True
+    else:
```

Comment:
I think this would be safer if you just checked for the presence of the thing and returned `False` otherwise, without try/catch; I don't like catching errors so broadly, as it seems like it'd be easy to hide e.g. a file reading issue.